apps/web/src/views/AddLiquidityV3/components/AprCalculator.tsx

```diff
@@ -158,13 +158,24 @@ export function AprCalculator({
   const validAmountA = amountA || (inverted ? tokenAmount1 : tokenAmount0) || (inverted ? aprAmountB : aprAmountA)
   const validAmountB = amountB || (inverted ? tokenAmount0 : tokenAmount1) || (inverted ? aprAmountA : aprAmountB)
   const [amount0, amount1] = inverted ? [validAmountB, validAmountA] : [validAmountA, validAmountB]
-  const inRange = isPoolTickInRange(pool, tickLower, tickUpper)
+  const inRange = isPoolTickInRange(pool ?? undefined, tickLower ?? 0, tickUpper ?? 0)
```

Comment:
The reason to accept `undefined` as tick is because `0` tick is valid and has meaning so it would be better to use undefined instead
```suggestion
  const inRange = isPoolTickInRange(pool ?? undefined, tickLower ?? undefined, tickUpper ?? undefined)
```